### DISCUSSION TITLE
Gridcoin <> CoopHive Partnership Proposal

### DISCUSSION THREAD

**Discussion by levirybalov** [gridcoin/Gridcoin-Research#2764] [General]
Forum to discuss https://www.reddit.com/r/gridcoin/comments/1ctuee7/hey_everyone_its_been_a_while/

**Comment by div72** [comment]
This is should be an issue in [gridcoin-community/Gridcoin-Tasks](https://github.com/gridcoin-community/Gridcoin-Tasks) rather than a discussion here in my opinion.

**Reply by Weber462** [reply]
I opened new discussion in suggested location.
